TestFoundation/TestProcessInfo.swift

```diff
@@ -71,7 +73,8 @@ class TestProcessInfo : XCTestCase {
 #if os(Windows)
         func setenv(_ key: String, _ value: String, _ overwrite: Int) -> Int32 {
           assert(overwrite == 1)
-          return putenv("\(key)=\(value)")
+          guard !key.contains("=") else { return -1 }
```

Comment:
Can you have `=` in the key on Unix?

Reply:
No, it will return EINVAL.

Reply:
Should `setenv` in that case behave similar to Unix and return `EINVAL`?

Reply:
I've made it match Unix and return return -1 and set `errno` to `EINVAL`.